### ISSUE TITLE
Safari 6 popup

### ISSUE THREAD

**Issue by sboc** [KangoExtensions/kango#3]
In Safari 6 - create an extension with a popover and install it.

Launch Safari 6 and click the red X to close/hide it. Open it again. Now popups no longer work.

**Comment by sboc** [comment]
I've actually found a workaround for this.

```
kango.browser.addEventListener(kango.browser.event.BeforeNavigate, function(event) {
    kango.ui.browserButton.setPopup(null);
    kango.ui.browserButton.setPopup({ url: url, width: 300, height: 300 });
});
```

seems to do the trick for now, however, does not work with multiple windows.

**Comment by akrylysov** [comment]
Fixed in 0.9.8
